app/containers/List/ListItem.js

```diff
@@ -86,7 +86,7 @@ const Button = React.memo(({
 }) => (
 	<Touch
 		onPress={() => onPress(props.title)}
-		style={{ backgroundColor: themes[props.theme].backgroundColor }}
+		style={{ backgroundColor: props.backgroundColor || themes[props.theme].backgroundColor }}
```

Comment:
Name the prop on params just like `onPress`, so we can use `backgroundColor` instead of `props.backgroundColor`.